frontend/src/app/main/ui/workspace/sidebar/options/shapes/group.cljs

```diff
@@ -49,7 +49,7 @@
 
     [:div.options
      [:& measures-menu {:type type :ids measure-ids :values measure-values :shape shape}]
-     [:& component-menu {:ids comp-ids :values comp-values}]
+     [:& component-menu {:ids comp-ids :values comp-values :shape shape}]
```

Comment:
Don't send down the whole shape if you only need the name, send directly the name.

Passing the whole shape down to the next component will make it rerender on all shape changes, but only name is needed.